src/telliot_feeds/sources/blockhash_aggregator.py

```diff
@@ -190,14 +192,30 @@ def parse_user_val(self) -> int:
         self.timestamp = data
         return data
 
+    def is_valid_timestamp(self, timestamp: int) -> bool:
+        """Check if timestamp is valid."""
+        try:
+            # Checking if timestamp is valid first
+            _ = datetime.fromtimestamp(timestamp)
+            # Timestamp should >= ethereum genesis block timestamp
+            if timestamp >= 1438269973:
```

Comment:
Should you also return `False` if the timestamp is greater than the current unix timestamp??

Reply:
I just added a check for this